demo/Makefile

```diff
@@ -0,0 +1,78 @@
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := default
+BOOTSTRAP_LOG_FILE := bootstrap.${CLUSTER}.log
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+
+.PHONY: install-cli init bootstrap up down halt update ssh reload token destroy dashboard ssh-cmd
+
+
+# Install flynn cli on host
+install-cli:
+	L=/usr/local/bin/flynn && curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >$L && chmod +x $L
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee ${BOOTSTRAP_LOG_FILE}
+	cat ${BOOTSTRAP_LOG_FILE} | awk '/flynn cluster add/' | awk 'BEGIN {RS=""}{gsub(/\r/,"",$$0); print $$0}' | sh -s
```

Comment:
We prefer using `bash` rather than `sh`, and `grep` for matching, so I suggest:

```
grep '^flynn cluster add' ${BOOTSTRAP_LOG_FILE} | bash
```

It may also be worth exiting non-zero if bootstrapping fails, e.g.:

```
(grep '^flynn cluster add' bootstrap.log || echo exit 1) | bash
```